packages/react-instantsearch-hooks-dom/src/widgets/Breadcrumb.tsx

```diff
@@ -36,6 +36,10 @@ export function Breadcrumb({
       items={items}
       hasItems={canRefine}
       onNavigate={refine}
+      translations={{
+        root: 'Home',
+        separator: '>',
```

Comment:
do we allow these to be overridden via props?

Reply:
I don't think Home should be overridden until we have a public translations API. For `separator` I'm not sure what's the best way to handle it.

What I understand is, there are 2 "separator" props:
- one is for the hook, to specify which character is used as a separator in the facet values
- another is for the UI

I initially put the UI separator in the translations, but stricly speaking it should not be there. Is it fine like this?
Alternatively we could default the separator prop used for the hook and stop exposing it (since we can be opinionated in our default widget implementation), and instead use the prop for the UI separator. 

Wdyt?

Reply:
I'm ok with initially having no way to customise it then

Reply:
Done in 7314cce